src/components/Reports/PeopleReport/PeopleReport.jsx

```diff
@@ -341,321 +342,321 @@ class PeopleReport extends Component {
-
-//         {/*        {priorityList.length > 0 ?*/}
-//         {/*            <Col class="block">*/}
-//         {/*                <Col>*/}
-//         {/*                  Priority:*/}
-//         {/*                </Col>*/}
-//         {/*                <Col>*/}
-//         {/*                  <ToggleButtonGroup type="checkbox" variant="info">*/}
-//         {/*                  {priorityList.map((c, index) => (*/}
-//         {/*                      <ToggleButton variant="info">{c}</ToggleButton>*/}
-//         {/*                  ))}*/}
-//         {/*                  </ToggleButtonGroup>*/}
-//         {/*                </Col>*/}
-//         {/*            </Col>*/}
-//         {/*          : <></>}*/}
-
-//         {/*        {statusList.length > 0 ?*/}
-//         {/*          <Col class="block">*/}
-//         {/*            <Col>*/}
-//         {/*              Status:*/}
-//         {/*            </Col>*/}
-//         {/*            <Col>*/}
-//         {/*              <ToggleButtonGroup type="checkbox" variant="info">*/}
-//         {/*                {statusList.map((c, index) => (*/}
-//         {/*                  <ToggleButton variant="info">{c}</ToggleButton>*/}
-//         {/*                ))}*/}
-//         {/*              </ToggleButtonGroup>*/}
-//         {/*            </Col>*/}
-//         {/*          </Col>*/}
-//         {/*        : <></>}*/}
-
-//         {/*        {classificationList.length > 0 ?*/}
-//         {/*          <Col class="block">*/}
-//         {/*            <Col>*/}
-//         {/*              Classification:*/}
-//         {/*            </Col>*/}
-//         {/*            <Col>*/}
-//         {/*              <ToggleButtonGroup type="checkbox" variant="info">*/}
-//         {/*                {classificationList.map((c, index) => (*/}
-//         {/*                  <ToggleButton variant="info">{c}</ToggleButton>*/}
-//         {/*                ))}*/}
-//         {/*              </ToggleButtonGroup>*/}
-//         {/*            </Col>*/}
-//         {/*          </Col>*/}
-//         {/*        : <></>}*/}
-
-//         {/*        {users.length > 0 ?*/}
-//         {/*          <Col class="block">*/}
-//         {/*          <Col>*/}
-//         {/*            User:*/}
-//         {/*          </Col>*/}
-//         {/*          <Col>*/}
-//         {/*            <ToggleButtonGroup type="checkbox" variant="info">*/}
-//         {/*                <ToggleButton variant="info">{users}</ToggleButton>*/}
-//         {/*            </ToggleButtonGroup>*/}
-//         {/*          </Col>*/}
-//         {/*          </Col>*/}
-//         {/*        : <></>}*/}
-//         {/*    </div>*/}
-//         {/*</Row>*/}
-//         {/*  <Row>*/}
-//         {/*     <ShowTasksCollapse userTaskList={userTaskList}/> /!*give margin zero on left & right to prevent cutting the edge *!/*/}
-//         {/*  </Row>*/}
-//       </>
-//       )
-//     }
+    //     const UserTask = (props) => {
```

Comment:
What's the point of keeping these commented out blocks here? if we aren't using this code.

Reply:
In fact, I just don't know if anyone needs this code:) Looks like someone commented it long ago. This happens in many places here, I don't know why